### PR TITLE
Sales receipt var

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
Changes in this branch include:

- Addition of the `using_sales_receipt` variable within the sales receipt [doulbe_entry_transaction](https://github.com/fivetran/dbt_quickbooks/compare/sales_receipt_var?expand=1#diff-148e1142c81fa7f76750343d47efcbd307d914ea788773097c184a939b320dbd), [sales_union](https://github.com/fivetran/dbt_quickbooks/compare/sales_receipt_var?expand=1#diff-093304d8185b50b927b1559842aad6a4dc29f318f6ea96e07d80158b6dada200), [expenses_sales_enhanced](https://github.com/fivetran/dbt_quickbooks/compare/sales_receipt_var?expand=1#diff-ca525231e5a9f9d23b077ecd44d435d0fea808e8348c2595516402b1636538c5), and [general ledger](https://github.com/fivetran/dbt_quickbooks/compare/sales_receipt_var?expand=1#diff-b9da735ae74375807d38663eff49ded18305432673b81dcb6d40f248693dca16) models.
- Inclusion of the `enabled_vars_one_true` macro in the [int_quickbooks__sales_union](https://github.com/fivetran/dbt_quickbooks/compare/sales_receipt_var?expand=1#diff-093304d8185b50b927b1559842aad6a4dc29f318f6ea96e07d80158b6dada200) model where the model will still run if at least one variable is true for `sales receipt` and `invoice`. Otherwise the model will not run.
- James Ayoub recommended changes to capture all customer and account id's within the transaction_line models for `bills`, `invoices`, `purchases`, and `vendor credits`.
- Minor format change within `refund receipt` transaction lines model.
- Temporary source package reference. Once the [PR](https://github.com/fivetran/dbt_quickbooks_source/pull/7) for the quickbooks source package has been approved this will be updated to the proper source version.

[Joe PR Loom Description](https://www.loom.com/share/7ec2459903834725a2b5f677b76024d3)

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 packages:
-  - package: fivetran/quickbooks_source
-    version: 0.1.1
\ No newline at end of file
+  # - package: fivetran/quickbooks_source
```
Comment (fivetran-kristin) [inline packages.yml:2]: revert this change prior to merging

**Review by fivetran-kristin — APPROVED** [review]
LGTM - you will need to remove the packages.yml file reference prior to merging. Thanks!

File: packages.yml (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply packages.yml]: Updated, thanks!

**Comment by fivetran-joemarkiewicz** [comment]
Made all the requested updates! Additionally, found a bug with disabling estimates failed the `quickbooks__ap_ar_enhanced` model. Updated the model to correctly run with estimates disabled.
